operate/importer-8_6/src/main/java/io/camunda/operate/zeebeimport/processors/JobZeebeRecordProcessor.java

```diff
@@ -59,26 +59,34 @@ public void processJobRecords(
       final boolean concurrencyMode)
       throws PersistenceException {
     LOGGER.debug("Importing Job records.");
-    for (final List<Record<JobRecordValue>> jobRecords : records.values()) {
-      processLastRecord(
-          jobRecords,
-          JOB_EVENTS,
-          rethrowConsumer(
-              record -> {
-                final JobRecordValue recordValue = (JobRecordValue) record.getValue();
-                processJob(record, recordValue, batchRequest, concurrencyMode);
-              }));
+    for (final List<Record<JobRecordValue>> flowNodeJobRecords : records.values()) {
+      final Map<Long, List<Record<JobRecordValue>>> groupedByJobId =
```

Comment:
One minor thing: the var is called `groupedByJobId` and it has `jobKeys` right? 
I'm mention this because we had some confusions in the past about keys and ids.

Reply:
Yes, it's grouped by jobKeys. I changed the naming.